src/ohri-form.component.test.tsx

```diff
@@ -59,290 +70,465 @@ jest.mock('@openmrs/esm-framework', () => {
     getAsyncLifecycle: jest.fn(),
     usePatient: jest.fn().mockImplementation(() => ({ patient: mockPatient })),
     registerExtension: jest.fn(),
-    useSession: jest.fn().mockImplementation(() => mockSessionDataResponse.data),
-    openmrsFetch: jest.fn().mockImplementation(args => mockOpenmrsFetch(args)),
+    useSession: jest
+      .fn()
+      .mockImplementation(() => mockSessionDataResponse.data),
+    openmrsFetch: jest
+      .fn()
+      .mockImplementation((args) => mockOpenmrsFetch(args)),
   };
 });
 
-jest.mock('../src/api/api', () => {
-  const originalModule = jest.requireActual('../src/api/api');
+jest.mock("../src/api/api", () => {
+  const originalModule = jest.requireActual("../src/api/api");
 
   return {
     ...originalModule,
-    getPreviousEncounter: jest.fn().mockImplementation(() => Promise.resolve(null)),
-    fetchConceptNameByUuid: jest.fn().mockImplementation(() => Promise.resolve(null)),
+    getPreviousEncounter: jest
+      .fn()
+      .mockImplementation(() => Promise.resolve(null)),
+    fetchConceptNameByUuid: jest
+      .fn()
+      .mockImplementation(() => Promise.resolve(null)),
     getConcept: jest.fn().mockImplementation(() => Promise.resolve(null)),
-    getLatestObs: jest.fn().mockImplementation(() => Promise.resolve({ valueNumeric: 60 })),
+    getLatestObs: jest
+      .fn()
+      .mockImplementation(() => Promise.resolve({ valueNumeric: 60 })),
+      saveEncounter: jest.fn(),
   };
 });
 
-describe('OHRI Forms:', () => {
+describe("OHRI Forms:", () => {
   afterEach(() => {
     cleanup();
     jest.useRealTimers();
   });
-
-  it('Should render by the form json without dying', async () => {
+const encounterContext: EncounterContext = {
+  patient: {
+    id: '833db896-c1f0-11eb-8529-0242ac130003',
+  },
+  location: {
+    uuid: '41e6e516-c1f0-11eb-8529-0242ac130003',
+  },
+  encounter: {
+    uuid: '873455da-3ec4-453c-b565-7c1fe35426be',
+    obs: [],
+  },
+  sessionMode: 'enter',
+  encounterDate: new Date(2023, 8, 29),
+  setEncounterDate: value => {},
```

Comment:
@lucyjemutai, are you using the encounter context in any of the tests?

Reply:
i am not using the encounterContext